index.html

```diff
@@ -53,6 +53,191 @@ <h4>I’am a Software Developer</h4>
+                                <span>Ruby</span>
+                            </div>
+                        </div>
+                        <button class="btn btn-card" type="button" >See Project</button>   
+                    </li>            
+                    <li class="project-card" >                       
+                        <div class="card-description" >
+                            <h1>Profesional Art Printing Data</h1>
+                            <p>A daily selection of privately personalized reads; no accounts or sign-ups required. 
+                                has been the industry's standard
+                            </p>
+                            <div>
+                                <span>html</span>
+                                <span>Bootstrap</span>
+                                <span>Ruby</span>
+                            </div>
+                        </div>
+                        <button class="btn btn-card" type="button" >See Project</button>   
+                    </li>            
+                    <li class="project-card" >                       
+                        <div class="card-description" >
+                            <h1>Profesional Art Printing Data</h1>
+                            <p>A daily selection of privately personalized reads; no accounts or sign-ups required. 
+                                has been the industry's standard
+                            </p>
+                            <div>
+                                <span>html</span>
+                                <span>Bootstrap</span>
+                                <span>Ruby</span>
+                            </div>
+                        </div>
+                        <button class="btn btn-card" type="button" >See Project</button>   
+                    </li>            
+                    <li class="project-card" >                       
+                        <div class="card-description" >
+                            <h1>Profesional Art Printing Data</h1>
+                            <p>A daily selection of privately personalized reads; no accounts or sign-ups required. 
+                                has been the industry's standard
+                            </p>
+                            <div>
+                                <span>html</span>
+                                <span>Bootstrap</span>
+                                <span>Ruby</span>
+                            </div>
+                        </div>
+                        <button class="btn btn-card" type="button" >See Project</button>   
+                    </li>            
+
+                </ul>               
+            </div>
+        </section>
+        <div id="about-and-skill-section" >
+            <section id="about-section">
+                <header class="header" >
+                    <h3 id="about-me-title">About me</h3>
+                </header>
```

Comment:
- [ ] [Optional][Highly Recommended] same here It would be better for search engine optimization to have one header per page, So please remove this `<header>` tag